src/api.js

```diff
@@ -4,8 +4,8 @@ const ncNewsAPI = axios.create({
   baseURL: "https://barry-nc-news.onrender.com/api/",
 });
 
-const availableArticles = () => {
-  return ncNewsAPI.get("articles").then((response) => response.data.articles);
+const availableArticles = (topic) => {
+  return ncNewsAPI.get("articles", {params: {topic}}).then((response) => response.data.articles);
```

Comment:
This is the type of dynamism I'm looking for! How can we make one component that's capable of calling this function and passing in whichever topic is currently being asked for? Hint: have a read up on useQuery